email_templates/carpeEmail/style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: red;
+}
```

Comment:
the inline styling seems to work fine, I was able to get the email font to be red and shiz like that for instance. Not sure if that's what you're referring to. Shippin.